lib/relations/belongsToOne/BelongsToOneUnrelateOperation.js

```diff
@@ -11,6 +11,27 @@ class BelongsToOneUnrelateOperation extends BelongsToOneRelateOperation {
     this.ids = [ids];
     return true;
   }
+
+  queryExecutor(builder) {
+    // NOTE: This is identical to BelongsToOneRelateOperation.queryExecutor()
```

Comment:
We could also put a part of this code into a helper function and reuse it?

Reply:
Nah, don't bother. It's not that many lines